x509-ocsp/src/ext.rs

```diff
@@ -0,0 +1,221 @@
+//! OCSP Extensions
+
+use alloc::vec::Vec;
+use const_oid::{
+    db::rfc6960::{
+        ID_PKIX_OCSP_ARCHIVE_CUTOFF, ID_PKIX_OCSP_CRL, ID_PKIX_OCSP_NONCE,
+        ID_PKIX_OCSP_PREF_SIG_ALGS, ID_PKIX_OCSP_RESPONSE, ID_PKIX_OCSP_SERVICE_LOCATOR,
+    },
+    AssociatedOid,
+};
+use der::{
+    asn1::{GeneralizedTime, Ia5String, ObjectIdentifier, OctetString, Uint},
+    Sequence, ValueOrd,
+};
+use spki::AlgorithmIdentifierOwned;
+use x509_cert::{
+    ext::{pkix::AuthorityInfoAccessSyntax, Extension},
+    impl_newtype,
+    name::Name,
+};
+
+#[cfg(feature = "rand_core")]
```

Comment:
wait, we don't need to explicitly declare the feature in the `Cargo.toml`? :exploding_head:

Reply:
Yeah, features and optional dependencies are somewhat confusingly merged into the same namespace, which is annoying because every optional dependency becomes a feature whether you want it to be or want to hide it as an implementation detail.

The `dep:` syntax somewhat fixes that.